src/NuGet.Protocol.Catalog/Models/PackageType.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using Newtonsoft.Json;
+
+namespace NuGet.Protocol.Catalog
+{
+    /// <summary>
+    /// Source: https://docs.microsoft.com/en-us/nuget/api/catalog-resource#catalog-leaf
+    /// </summary>
+    public class PackageType
+    {
+        [JsonProperty("@id")]
+        public string Url { get; set; }
```

Comment:
Nothing but the `name` property are necessary. I would prefer excluding `@id` and `@type` since they are very stupid values. `version` is fine albeit only forward looking.